Assets/HoloToolkit-Examples/UX/Scripts/GestureInteractive.cs

```diff
@@ -266,8 +263,11 @@ public override void OnFocusEnter()
         private Vector3 GetCurrentHandPosition()
         {
             Vector3 handPosition;
+#if UNITY_2017_2_OR_NEWER
```

Comment:
When are we going to stop adding these checks for Unity versions toolkit branch does not support?
I'm not a fan of changes like these which force people to update scripts for no benefit :(

Reply:
Because there's lots of Devs who still use the toolkit that need the support for older editors